client/web/src/repo/RepoHeader.tsx

```diff
@@ -204,7 +204,11 @@ export const RepoHeader: React.FunctionComponent<React.PropsWithChildren<Props>>
                 <Breadcrumbs
                     breadcrumbs={props.breadcrumbs}
                     location={props.location}
-                    className={classNames('justify-content-start', !props.forceWrap ? styles.breadcrumbWrap : '')}
+                    className={classNames(
+                        'justify-content-start',
+                        styles.breadcrumb,
```

Comment:
```suggestion
                        'justify-content-start pr-0',
```
nit: as an alternative

Reply:
This doesn't work because there's already a `pr-2` added inside the Breadcrumb component that takes precedence :( I'll need to audit where Breadcrumbs are called from

Reply:
Okay I’m stupid, there's only one caller 😓 Fixing inside `<Breadcrumbs/>` instead